pkgs/applications/window-managers/wmderland/default.nix

```diff
@@ -0,0 +1,51 @@
+{ stdenv, fetchFromGitHub, cmake, libnotify, libX11, xorgproto, nixosTests }:
+
+with stdenv.lib;
```

Comment:
Stdenv.lib is deprecated as per recent contributing guide change. Also please only use with in the smallest possible scope.